cg/cli/demultiplex/demux.py

```diff
@@ -80,20 +80,18 @@ def demultiplex_all(context: CGConfig, flow_cells_directory: click.Path, dry_run
 @click.command(name="flow-cell")
 @click.argument("flow-cell-name")
 @DRY_RUN
-@OPTION_BCL_CONVERTER
 @click.pass_obj
 def demultiplex_flow_cell(
     context: CGConfig,
     dry_run: bool,
     flow_cell_name: str,
-    bcl_converter: str,
 ):
-    """Demultiplex a flow cell.
+    """Demultiplex a flow cell using BclConverter.

```

Comment:
```suggestion
    """Demultiplex a flow cell using BCLConvert.
```
"BCL Convert" is the software name